library/socket/addrinfo/initialize_spec.rb

```diff
@@ -53,7 +53,7 @@
         @addrinfo.ip_port.should == 25
       end
 
-      it "returns the Socket::UNSPEC pfamily" do
+      it "returns the INET6 pfamily" do
```

Comment:
Looks good, but in most cases a test checks that a getter returns a value passed into a constructor and nothing more. So a test title may be more generic, e.g. `returns pfamily`. And it seems it doesn't matter what a specific value is used.